### ISSUE TITLE
export via API

### ISSUE THREAD

**Issue by vielmetti** [localwiki/localwiki#752]
Looking to get some requirements settled for an export format for Localwiki.

I'd like to get a weekly dump of Arborwiki in some well-specified format to upload to the Ann Arbor District Library as an off-site backup. I think this can all be done via the API. Ideally I'd like to bake the code that does this into Localwiki itself, so that anyone with the proper rights can push out an export of their area.

thanks

**Comment by amkelly** [comment]
I would also love to see this added. As vielmetti says, it looks like this might be possible to do with the API, but it is essential that communities who are able to take advantage of localwiki can be assured that there's a second copy of all their work held locally. I'm also curious to see if anyone has successfully set up backups or mirroring with the API.

**Comment by philipn** [comment]
Yes, it's very easy to do an export using the API.  You would just iterate through each of the API endpoints, filtering on the region id parameter.  Then just save the resulting JSON somewhere.  You might also want to download each of the images linked in the `files` endpoint.

**Comment by reidserozi** [comment]
@vielmetti @amkelly Did you by chance follow through on @philipn suggestion? Same thought here from Raleigh LocalWiki. Thank you

**Comment by vielmetti** [comment]
@amkelly @reidserozi I still want to do this but haven't had the opportunity to yet.
